identity-service/src/notifications/sendNotifications/publishNotifications.js

```diff
@@ -32,11 +32,17 @@ const getPublishNotifBaseType = (notification) => {
       return notificationTypes.Create.base
     case notificationTypes.ChallengeReward:
       return notificationTypes.ChallengeReward
+    case notificationTypes.MilestoneListen:
+    case notificationTypes.MilestoneFavorite:
+    case notificationTypes.MilestoneFollow:
+    case notificationTypes.MilestoneRepost:
+      return notificationTypes.Milestone
   }
 }
 
 const solanaNotificationBaseTypes = [
-  notificationTypes.ChallengeReward
+  notificationTypes.ChallengeReward,
+  notificationTypes.MilestoneListen
```

Comment:
Do you only want to add MilestoneListen here, or `notificationTypes.Milestone`?

Reply:
Just milestone listen because the other milestones are not yet handled by solana